java/maven/external/asm-7.0-license.txt

```diff
@@ -0,0 +1,42 @@
+Name: ASM
+Description: ASM is an all purpose Java bytecode manipulation and analysis framework
+Origin: https://asm.ow2.io/
+Version: 7.0
+License: ASM
+URL: https://asm.ow2.io/
+Files: asm-7.0.jar, asm-commons-7.0.jar
+Type: compile-time
+Comment: Used to build ueber-jar bundling for example json library into NbEventSpy
```

Comment:
@matthiasblaesing did you mean _simple_ json instead of _example_?

Reply:
Yes - and to answer the other question: feel free to squash.